src/commands/General/mergepdf.ts

```diff
@@ -0,0 +1,42 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import type {
+ Args, CommandOptions, Result, UserError,
+} from '@sapphire/framework';
+import type { Message } from 'discord.js';
+import PdfMerger from 'pdf-merger-js';
+import { mergePDF as config } from '@/config/commands/general';
+import MonkaCommand from '@/structures/MonkaCommand';
+import type { GuildMessage } from '@/types';
+
+@ApplyOptions<CommandOptions>(config.options)
+export default class PdfMergeCommand extends MonkaCommand {
+  public async run(message: GuildMessage, args: Args): Promise<void> {
+    // Argument dans l'ordre croissant
```

Comment:
Que veux dire ce commentaire 🤔

Reply:
Qu'il faut mettre les fichiers dans l'ordre croissant, je pense créer deux options:
- On définit l'ordre des fichiers à fusionner
- On laisse le bot définir l'ordre en utilisant un tri

Reply:
pourquoi ne pas juste garder l'ordre des attachments et des messages ?

Reply:
J'y ai pas pensé !

Reply:
😄 
du coup on peut laisser comme ca, les messages vont être récupéré dans l'ordre donné, et les attachments vont être itérés dans l'ordre où ils ont été envoyés

Reply:
(du coup tu peux supprimer le commentaire)

Reply:
```suggestion
```